motr/client_init.c

```diff
@@ -1600,9 +1600,15 @@ int m0_client_init(struct m0_client **m0c_p,
 
 	if (conf->mc_is_addb_init) {
 		char buf[64];
+		/* Default client addb record file size set to 128M */
+		m0_bcount_t size = 128ULL << 20;
```

Comment:
This was the format I suggested.

Some other people may prefer:
m0_bcount_t size = 128ULL * 1024 * 1024;

This is another option.

Reply:
Acked...But kept the 128ULL << 20 format